docs/testbed/README.testbed.WANSetup.md

```diff
@@ -30,33 +30,64 @@ First, we need to prepare the host where we will be configuring the virtual test
 ## Download an VM image
 We currently support IOS-XR-based, EOS-based or SONiC VMs to simulate neighboring devices in the virtual testbed, much like we do for physical testbeds. To do so, we need to download the image to our testbed host.
 
-### Option 1: cEOS (container-based) image (experimental)
-#### Option 1.1: Download and import cEOS image manually
-1. Download the [cEOS image from Arista](https://www.arista.com/en/support/software-download)
-2. Import the cEOS image (it will take several minutes to import, so please be patient!)
-
-```
-docker import cEOS-lab-4.25.5.1M.tar.xz ceosimage:4.25.5.1M
-```
-After imported successfully, you can check it by 'docker images'
-```
-$ docker images
-REPOSITORY                                             TAG           IMAGE ID       CREATED         SIZE
-ceosimage                                              4.25.5.1M     fa0df4b01467   9 seconds ago   1.62GB
-```
-**Note**: *For time being, the image might be updated, in that case you can't download the same version of image as in the instruction,
-please download the corresponding version(following [Arista recommended release](https://www.arista.com/en/support/software-download#datatab300)) of image and import it to your local docker repository.
-The actual image version that is needed in the installation process is defined in the file [ansible/group_vars/all/ceos.yml](../../ansible/group_vars/all/ceos.yml), make sure you modify locally to keep it up with the image version you imported.*
+### Option 1: cEOS (container-based) image (recommended)
 
-**Note**: *Please also notice the type of the bit for the image, in the example above, it is a standard 32-bit image. Please import the right image as your needs.*
+1. **Prepare folder for image files on test server**
+
+    Create subfolder `images` under `root_path` defined in the `ansible/group_vars/vm_host/main.yml` file. Assume `root_path` is `veos-vm`:
+
+    ```bash
+    mkdir -p ~/veos-vm/images
+    ```
+
+2. **Prepare the cEOS image file**
+
+   #### Option 1.1: Manually download cEOS image
+
+   1. Obtain the cEOS image from [Arista's software download page](https://www.arista.com/en/support/software-download).
+   2. Place the image file under subfolder `images` of the location determined by `root_path` variable in `ansible/group_vars/vm_host/main.yml`:
+
+      Assume `root_path` is `veos-vm`:
```

Comment:
Nit rephrase:

Assuming you set `root_path` to `veos-vm`, you should run the following command: